### PR TITLE
Frontend styling

### PR DESCRIPTION
**Pull request by Asamya**
Added the logo for the app and edit styling

### CODE REVIEW

File: frontend/src/components/commons/Searchbar.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Searchbar = (props) => {
 
 const InputStyled = styled.input`
   width: 80%;
```
Comment (benjamin-ranft) [inline frontend/src/components/commons/Searchbar.js:16]: would use a grid here too with 3 rows, the two outer ones being the spacing between the screen edge and the searchbar

File: frontend/src/components/commons/Searchbar.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const Searchbar = (props) => {
...
-  place-self: center;
+  place-self: end;
```
Comment (benjamin-ranft) [inline frontend/src/components/commons/Searchbar.js:17]: Could use "justify-self" as well

File: frontend/src/components/home/HomeAppBar.js [hunk 3]
```diff
@@ -8,9 +8,7 @@ export default function BottomAppBar() {
     const history = useHistory();
 
     return <HeaderStyled>
-            <h2>
-                Rubble
-            </h2>
+            <ImageStyled src='/rubble_logo.png' alt='rubble logo'/>
```
Comment (benjamin-ranft) [inline frontend/src/components/home/HomeAppBar.js:11]: Maybe make it clickable to bring user back to home screen?

File: frontend/src/components/user/UserAppBar.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ export default function UserAppBar() {
     const {logout} = useContext(UserContext);
 
     return <HeaderStyled>
-        <h2>Rubble</h2>
+            <ImageStyled src='/rubble_logo.png' alt='rubble logo' onClick={() => history.push('/dashboard')}/>
             <NavigationGroupStyled>
                 <NavLinkStyled to='/dashboard'><RiMailLine/></NavLinkStyled>
```
Comment (benjamin-ranft) [inline frontend/src/components/user/UserAppBar.js:22]: You don't need to wrap the icon in a nav link element. You can use "StyledICONNAME" and an onClick event too. (don't know if it makes it any better though 😅)

**Review by benjamin-ranft — APPROVED** [review]
Nice work!

File: frontend/src/components/user/UserAppBar.js (hunk 4 above)
Comment (Asamya) [reply frontend/src/components/user/UserAppBar.js:22]: Thanks for the hint but the three icons are actually a navigation. When I implement them as a button they need a function that the navbar already provide.

**Pull request merged**